### DISCUSSION TITLE
n8n queue mode and worker + webhook processor

### DISCUSSION THREAD

**Discussion by mredodos** [coollabsio/coolify#4150] [Service Template Requests]
Can you add a template with pre-configured queue mode and template for workers? 

https://docs.n8n.io/hosting/scaling/queue-mode/
https://docs.n8n.io/hosting/scaling/queue-mode/#webhook-processors

**Comment by lamurian** [comment]
Since there has been no reply, I'd love to nudge this discussion a bit. I don't have any technical background, and my goal is to horizontally scale my N8N service via Coolify. It'd be very nice if there were a pre-configured queue mode and template for workers. Would this be possible to achieve with Coolify?

**Reply by mredodos** [reply]
yes is possible to handle queue mode via coolify but is a little bit tricky to do manually

**Comment by mredodos** [comment]
it seems is relased in coolify
